src/module.c

```diff
@@ -1300,6 +1320,26 @@ int RM_BlockedClientMeasureTimeEnd(RedisModuleBlockedClient *bc) {
     return REDISMODULE_OK;
 }
 
+/* This API allows modules to let Redis process some commands during long
+ * blocking execution of a module command. The module can call this API
+ * periodically, and after the time defined by the `script-time-limit` config,
+ * Redis will start rejecting most commands with `-BUSY` error, but allow the
+ * ones marked with the `allow-busy` flag to be executed. This API can also be
+ * used in thread safe context (while locked), and during loading (in the
+ * rdb_load, in which case it'll reject commands with -LOADING error) */
+void RM_SlowContextHeartbeat(RedisModuleCtx *ctx) {
```

Comment:
I think some of the terms here are confusing - I propose to name this `RM_Yield`.

What exactly is a `SlowContext`? We didn't define this anywhere. And `Heartbeat` is usually used to describe a process where one tells another it's still alive - but this is not the case here (it's not *required* to call this function).

Reply:
I also like the name RM_Yield()